griptape/schemas/base_schema.py

```diff
@@ -186,3 +230,11 @@ def is_list_sequence(cls, field_type: type | _SpecialForm) -> bool:
                 return issubclass(field_type, Sequence)
         else:
             return False
+
+    @classmethod
+    def is_union(cls, field_type: type) -> bool:
+        return field_type is Union or get_origin(field_type) is Union
+
+    @classmethod
+    def is_enum(cls, field_type: type) -> bool:
+        return isinstance(field_type, type) and issubclass(field_type, Enum)
```

Comment:
Can you please move these methods (and `is_list_sequence`) above the protected methods?

Reply:
Ignore me here, my other comment said to make them protected. Continue with that.